docs/index.md

```diff
@@ -232,7 +278,8 @@ Open your projects `/ios/Podfile` and add any of the globals shown below to the
 
 ```ruby
 # Override Firebase SDK Version
-$FirebaseSDKVersion = '10.3.0'
+# Between `use_react_native!()` paragraph and `target 'RNFBTests' do ... end`
```

Comment:
It does not matter where it goes actually. It's just defining a variable in Ruby syntax, at the top level.

```suggestion
```